From 1804c7cb2e6e62a363a18f237ecdf8337e58c20d Mon Sep 17 00:00:00 2001
From: Martin Roth <martin@coreboot.org>
Date: Mon, 10 May 2021 11:28:45 -0600
Subject: [PATCH 06/19] src/security/intel/stm: Add warning for
 non-reproducible build

Because the STM build doesn't use the coreboot toolchain it's not
reproducible.  Make sure that's displayed during the build.

Signed-off-by: Martin Roth <martin@coreboot.org>
Change-Id: I3f0101400dc221eca09c928705f30d30492f171f
Reviewed-on: https://review.coreboot.org/c/coreboot/+/54020
Tested-by: build bot (Jenkins) <no-reply@coreboot.org>
Reviewed-by: Angel Pons <th3fanbus@gmail.com>
Reviewed-by: Patrick Georgi <pgeorgi@google.com>
---
 src/security/intel/stm/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/security/intel/stm/Makefile b/src/security/intel/stm/Makefile
index 1493869e80..31e5bdd88a 100644
--- a/src/security/intel/stm/Makefile
+++ b/src/security/intel/stm/Makefile
@@ -18,6 +18,8 @@ all: build
 
 build:
 	echo "STM - Build"
+	echo "--  WARNING: This uses the system toolchain instead of"
+	echo "    the coreboot toolchain, so is not reproducible."
 	cd $(project_dir)/Stm; \
 	mkdir -p build; \
 	cd build; \
-- 
2.25.1

